src/providers/DataProviders.test.js

```diff
@@ -0,0 +1,546 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import '../../test/jest/__mock__';
+
+import DataProvider from './DataProvider';
+
+const Children = () => (
+  <h1>DataProvider</h1>
+);
+
+const resources = {
```

Comment:
@KetineniM would you mind moving these resources under: https://github.com/folio-org/ui-inventory/tree/master/test/fixtures after that I think we can merge this!

Reply:
@mkuklis - moved resource under https://github.com/folio-org/ui-inventory/tree/master/test/fixtures